src/components/RouteOption/DockedEbikeRouteOption.jsx

```diff
@@ -10,14 +10,24 @@ export default function DockedEbikeRouteOption({
   onStartingPointStationChange,
   onDestinationStationChange,
 }) {
+  const {
+    leaveAt,
+    arriveAt,
+    takeBikeAt,
+    parkBikeAt,
+    travelTimeTotal,
+    walkingTimeFromStartingPoint,
+    cyclingTimeStationToStation,
+    walkingTimeToDestination,
+  } = routeOption.details;
```

Comment:
`routeOptions` can be `null`, in which case this currently fails (in fact, that's the reason why one of the tests is crashing currently)

```
    page.evaluate: StorybookTestRunnerError: 
    An error occurred in the following story. Access the link for full output:
    https://66c62ded00306661f2c5e455-dyisqudxbf.chromatic.com/?path=/story/components-routeoptioncontainer--default&addonPanel=storybook/interactions/panel

    Message:
     Cannot destructure property 'leaveAt' of 'routeOption.details' as it is undefined.
```

please refactor this code so that it still works and renders the docking station selectors without the route details when `routeOption.details` is null

you can use the `Default` story for testing this out. in its default state, it won't have any route details. only when it gets selected will `routeOption.details` become non-null and can we render those route details